src/flownet/ahm/_run_ahm.py

```diff
@@ -37,10 +37,8 @@ def _find_training_set_fraction(
 
     if config.flownet.training_set_end_date is not None:
         if config.flownet.training_set_fraction is not None:
-            print(
+            raise ValueError(
```

Comment:
Is this check necessary when failing earlier?

Reply:
In theory it should never get there.... but if one bypasses the config file and calls functions directly it will trigger. 

So, yeah, we could remove it if we only go via the config file route.